src/Services/LightningService.cs

```diff
@@ -368,7 +378,7 @@ await foreach (var response in openStatusUpdateStream.ResponseStream.ReadAllAsyn
                                     CreationDatetime = DateTimeOffset.Now,
                                     FundingTx = fundingTx,
                                     FundingTxOutputIndex = response.ChanOpen.ChannelPoint.OutputIndex,
-                                    BtcCloseAddress = closeAddress?.Address.ToString(),
```

Comment:
There's a possibility the closeAddress is null, we should handle it like the comment above.